geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/AuditoryCortexExample.js

```diff
@@ -0,0 +1,228 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core';
+import Canvas from '../../Canvas';
+import CameraControls from '../../../camera-controls/CameraControls';
+import Loader from "@geppettoengine/geppetto-ui/loader/Loader";
+import Button from "@material-ui/core/Button";
+
+const INSTANCE_NAME = 'acnet2';
+const COLORS = [
+  { r: 0, g: 0.2, b: 0.6, a: 1 },
+  { r: 0.8, g: 0, b: 0, a: 1 },
+  { r: 0, g: 0.8, b: 0, a: 1 },
+  { r: 0, g: 0.8, b: 0, a: 0.5 },
+];
+const SELECTION_COLOR = { r: 0.8, g: 0.8, b: 0, a: 1 };
+
+const styles = () => ({
+  container: {
+    height: '800px',
+    width: '1240px',
+    display: 'flex',
+    alignItems: 'stretch',
+  },
+});
+
+class AuditoryCortexExample extends Component {
+  constructor (props) {
+    super(props);
+    this.canvasRef = React.createRef();
+    this.state = {
+      hasModelLoaded: false,
+      showLoader: true,
+      data: [
+        {
+          instancePath: 'acnet2.baskets_12',
+          color: COLORS[1],
+        },
+        { instancePath: 'acnet2' },
+        {
+          instancePath: 'acnet2.baskets_12[0]',
+          color: COLORS[2],
+        },
+        {
+          instancePath: 'acnet2.baskets_12[7]',
+          color: COLORS[3],
+        },
+      ],
+      selected: {},
+      cameraOptions: {
+        angle: 60,
+        near: 10,
+        far: 2000000,
+        baseZoom: 1,
+        cameraControls: {
+          instance: CameraControls,
+          props: { wireframeButtonEnabled: false, },
+        },
+        reset: false,
+        autorotate: false,
+        wireframe: false,
+        position: { x: 230.357, y: 256.435, z: 934.238 },
+        rotation: { rx: -0.294, ry: -0.117, rz: -0.02, radius: 531.19 },
+      },
+    };
+
+    this.lastCameraUpdate = null;
```

Comment:
don't understand why we keep track of lastCameraUpdate here in this way, just curious, no change request here

Reply:
We (now) control the camera (position, rotation) through props (camOptions).
However, the movement in the scene doesn't go through the canvas component but rather affects the scene directly (using Threejs TrackballControls).
If we don't keep track of the cameraUpdates (using the camera handler) future renders of the canvas component will pass a past position in the camOptions props (making the camera jump to a previous place)

Reply:
Added this issue to #67